src/runtime/aot_executor/aot_executor.cc

```diff
@@ -132,6 +132,7 @@ PackedFunc AotExecutor::GetFunction(const std::string& name,
       *rv = this->GetInputIndex(args[0].operator String());
     });
   } else {
+    LOG(INFO) << "Unknown packed function: " << name;
```

Comment:
I'm looking for a way to show some output message for developers that they are using the wrong function name for this module. Specially since we have multiple version of this runtime.Module it would be very useful to have some messaging. What is your suggestion?
Also is LOG(DEBUG) better in this case?

Reply:
does [this](https://github.com/apache/tvm/blob/main/python/tvm/runtime/module.py#L152) work? or are you saying from c++?